applepay/utils.py

```diff
@@ -1,20 +1,40 @@
+"""
+The utilites in this module are used to verify and parse an apple pay token.  The
+utilities assume the token is a python dictionary.  See the #ApplePaySpec below for
+more documentation and guidelines on the steps taken here.
+
+#ApplePaySpec: https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html#//apple_ref/doc/uid/TP40014929-CH8-SW2
+"""
 import base64
-from datetime import timedelta
+import binascii
+from datetime import timedelta, datetime
+from functools import partial
+from itertools import ifilter
+import hashlib
 import logging
 
 from asn1crypto import cms
+from asn1crypto.parser import emit
```

Comment:
Would you mind consolidating this import with the one above it, so it's the following?

`from asn1crypto import cms, parser`
`parser.emit(...)'